zerver/webhooks/airbyte/view.py

```diff
@@ -0,0 +1,87 @@
+# Webhooks for external integrations.
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import webhook_view
+from zerver.lib.response import json_success
+from zerver.lib.typed_endpoint import JsonBodyPayload, typed_endpoint
+from zerver.lib.validator import WildValue, check_bool, check_int, check_string
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+AIRBYTE_TOPIC_TEMPLATE = "{workspace} - {connection} - {source} - {destination}"
+AIRBYTE_MESSAGE_TEMPLATE = (
+    "Job {jobId} {status} in {durationFormatted}. "
+    "Bytes Emitted: {bytesEmittedFormatted}, Bytes Committed: {bytesCommittedFormatted}. "
+    "Records Emitted: {recordsEmitted}, Records Committed: {recordsCommitted}. "
+    "Started at: {startedAt}, Finished at: {finishedAt}. "
+    "Bytes Emitted: {bytesEmitted}, Bytes Committed: {bytesCommitted}. "
+    "Duration In Seconds: {durationInSeconds}"
```

Comment:
Can you use Zulip's formatting to make this easier to read and more useful?

I don't know much about how Airbyte is used, but I'm pretty sure that we can remove useless/duplicate information. Finish time is almost always going to be now, so I'd be tempted to do something like:

`Job {jobId} succeeded in {DurationFormatted}." with maybe some details in a bulleted list.

https://docs.airbyte.com/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications#enable-the-slack-notification-in-airbyte

Has an example of what the Airbyte folks thought useful to put into a chat notification, and may be a useful guide.

Reply:
Thank you for your feedback. I'll work on the changes. Please, can you point me to Zulip's formatting sample? This is my first contribution to this codebase. It will be faster and easier for me if I know where to look. Thank you.

Reply:
You can look at how some other existing webhook notification templates are formatted. But mostly @timabbott was referring to Zulip's Markdown formatting for messages. So newlines, bulleted lists, bold/italic text, etc.

If you look at the screenshot you generated, it's hard to read the message as it's formatted. In the dev environment, you can go to /devtools/integrations and generate some messages from different existing integrations to see how the Zulip messages appear.

Reply:
Successfully refactored the code to make it more readable, implementing the following as pointed out:
* Formatted the responses using Zulip's Markdown formatting to make the responses more readable.
* Stripped out duplicate information from the response.